src/tsx/pdfViewer.tsx

```diff
@@ -63,6 +70,24 @@ class PdfViewer extends React.PureComponent<IProps, IState> {
     }
   }
 
+  private handleKey(event: KeyboardEvent) {
+    // Only accept keyboard shortcuts when nothing is focused
+    if (document.activeElement === document.body) {
+      if (event.key === "PageUp") {
+        this.scrollVertical(-100);
+      } else if (event.key === "PageDown") {
+        this.scrollVertical(100);
+      }
+    }
+  }
+
+  private scrollVertical(amount: number) {
+    const container = document.getElementById(this.state.id);
+    container.scrollBy({
+      top: amount,
```

Comment:
I think ```behavior: "smooth"``` would be nice here

Reply:
I used it first but it was troublesome when holding the key down. Without the smooth it works out of the box, with the smooth it resets the scroll during the animation and everything gets superslow. It might be possible to solve but felt like low prio.

Reply:
Must be another Chrome/Firefox difference then, it works fine for me... let's skip it then.